Dockerfile

```diff
@@ -15,7 +15,8 @@ RUN yum install -y redis \
     gcc-c++ \
     npm \
     wget \
-    totem
+    totem \
+    supervisor
```

Comment:
Its 3.0b3 as opposed to 3.1.3, but it seems to work just fine. I'm not sure its a big deal but it means that in my head - yum is all the libraries for running zamboni and pip are all the python deps the code needs.